src/Service/InstallationService.php

```diff
@@ -350,7 +350,7 @@ private function handleInstallationJson(string $bundle)
      */
     private function translateCoreReferences()
     {
-        foreach ($this->objects as $translateFrom => $value) {
+        foreach (array_keys($this->objects) as $translateFrom) {
```

Comment:
Why array_keys here?

Reply:
This only works if you do not use the $value parameter in the foreach
or if you do replace the use of $value with $this->objects[$translateFrom]

Reply:
A refactor so we dont have a unused variable $value because we only need the keys from $this->objects. So its prettier to loop through array_keys($array) than do for ($array as $key => $value) where value would never be used.

Reply:
Cool, I couldnt see enough to be sure $value wasn't used. If $value indeed is not used, I understand :).